src/style.css

```diff
@@ -1,3 +1,64 @@
+:root {
+  --blue: #00acdc;
+}
+
 body {
+  font-family: Roboto, Helvetica, Arial, sans-serif;
+  margin: 0;
+  background-color: aqua;
+  color: var(--blue);
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  background-color: white;
+}
+
+.logo {
+  margin: auto auto auto 50px;
+}
+
+.nav {
+  margin: auto 50px auto auto ;
```

Comment:
The same issue with the vertical margins here. Try to be consistent and add only top or only bottom margin.